src/main/resources/metainfo.gradle

```diff
@@ -193,8 +199,8 @@ allprojects {
                             if (isConfigSupported(conf)) {
                                 println "SRCLIB-DEPENDENCY $conf.name:$group:$name:$version:$file"
                             }
-                        } catch (Exception e) {
-                            println "SRCLIB-WARNING $e"
+                        } catch (e) {
+                            println "SRCLIB-WARNING 1 $e"
```

Comment:
Replaced with exception stack traces (I'd like to be able to understand the place where exception occurred)